docs/userguides/departingemployee.md

```diff
@@ -1,49 +1,28 @@
-# Search and Add user to Departing Employee List
+# Add or Remove user to Departing Employee List
```

Comment:
Not a big deal, but this title is a bit off... `Add or remove users from the Departing Employee list` is a little better I think

Reply:
Changed.